Campaigns/Ransomware hits healthcare - Possible compromised accounts.txt

```diff
@@ -0,0 +1,22 @@
+//Identify accounts that have logged on to affected endpoints
+// Check for specific alerts
+DeviceAlertEvents 
+// Attempts to clear security event logs.
+| where Title in("Event log was cleared", 
+// List alerts flagging attempts to delete backup files.
+"File backups were deleted", 
+// Potential Cobalt Strike activity - Note that other threat activity can also 
+//trigger alerts for suspicious decoded content
+"Suspicious decoded content", 
+// Cobalt Strike activity
+"\'Atosev\' malware was detected", 
+"\'Ploty\' malware was detected", 
+"\'Bynoco\' malware was detected")
+| extend AlertTime = Timestamp
+| distinct DeviceName, AlertTime, AlertId, Title
+| join DeviceLogonEvents on $left.DeviceName == $right.DeviceName

```

Comment:
no need to specify left and right when the column name is the same, you can just use:
| join DeviceLogonEvents on DeviceName